recipe/meta.yaml

```diff
@@ -1,50 +1,50 @@
 {% set name = "pynndescent" %}
-{% set version = "0.5.4" %}
+{% set version = "0.5.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 221124cbad8e3cf3ed421a4089d80ac5a29d3215e76cb49effc1df887533d2a8
+  sha256: 5d5dc683c03ef55fe3ddf693859720ca18f85c6e6e5bb0b4f14870278d5288ad
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # numba and llvmlite aren't available on s390x
+  skip: true  # [py<36 or s390x]
```

Comment:
```suggestion
  skip: true  # [py<37 or s390x]
```

Reply:
Maybe we should change this because 3.6 is deprecated. See [here](https://github.com/lmcinnes/pynndescent/blob/0.5.10/appveyor.yml)

Reply:
I follow the upstream https://github.com/lmcinnes/pynndescent/blob/cb5fa2e067cf35ffada66957217a0f3de504cc2c/setup.py#L26

Reply:
I don't think that actually does anything. I think we should follow [this](https://github.com/lmcinnes/pynndescent/blob/cb5fa2e067cf35ffada66957217a0f3de504cc2c/appveyor.yml#L5)

Reply:
Fixed as py<37